src/gui/editorwidgets/qgsvaluemapconfigdlg.cpp

```diff
@@ -402,11 +404,11 @@ void QgsValueMapConfigDlg::loadMapFromCSV( const QString &filePath )
       ceils << match.capturedTexts().last().trimmed().replace( QLatin1String( "\"\"" ), QLatin1String( "\"" ) );
     }
 
-    if ( ceils.size() != 2 )
+    if ( ceils.size() == 0 )
```

Comment:
@agiudiceandrea 
Since the `cells.size() > 2` is gone, just wanted to ask you if that's on purpose or if it's still safe.

Reply:
Thanks for the review!
If my logic isn't faulty, the QStringList `ceils` is populated in a `while ( matches.hasNext() && ceils.size() < 2 )` loop, so it cannot have a size greater than 2: its size can only be either 0 or 1 or 2.